openmls/src/treesync/diff.rs

```diff
@@ -835,8 +913,29 @@ impl<'a> TreeSyncDiff<'a> {
             .collect()
     }
 
-    /// Get the length of the direct path of the given [`LeafNodeIndex`].
-    pub(super) fn direct_path_len(&self, leaf_index: LeafNodeIndex) -> usize {
-        self.diff.direct_path(leaf_index).len()
+    /// Returns the filtered common path two leaf nodes share. If the leaves are
+    /// identical, the common path is empty.
```

Comment:
When the leaves are identical, shouldn't the common path be the leaf's direct path?